src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -140,8 +146,19 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
     String messageContent = request.getParameter("message");
 
-    // this removes any HTML from the message content
-    String cleanedMessageContent = Jsoup.clean(messageContent, Whitelist.none());
+    // this removes the new line from forming before acceptable html tags: b, em, i, u, strong
+    OutputSettings settings = new OutputSettings();
+    settings.prettyPrint(false);
+
+    // this removes any style / script / html (other than b, em, i, strong, u)
+    // from the message content
+    String cleanedMessageContent = Jsoup.clean(messageContent, "", Whitelist.simpleText(), settings);
+
+    // this will be used to use markdown instead of html to style text
```

Comment:
If you plan to uncomment this at a later stage, leave a small note about why its commented for now.

Reply:
okay will do!